README.md

```diff
@@ -86,19 +82,14 @@ Function | Description
 `isAuthenticated` | returns true if the user is not anonymous.
 `isFullyAuthenticated` | returns true if the is not an anonymous or a remember-me user.
 
-Additional information in [#29](https://github.com/joinfaces/joinfaces/issues/29)
-
 ## System Requirements
 
-Joinfaces | Java | Spring Boot | JSF
```

Comment:
I think we should keep this table in the readme

Reply:
JoinFaces versions table restored. Link Key Features to Reference Guide.